clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -160,6 +160,11 @@ New checks
   Replaces nested ``std::min`` and ``std::max`` calls with an initializer list
   where applicable.
 
+- New :doc:`cppcoreguidelines-pro-bounds-avoid-unchecked-container-accesses
```

Comment:
it it in line 150

Reply:
Are you referring to a double "it" somewhere? I am not seeing the error in line 163 or line 150?